samples/react-native/src/Screens/ErrorsScreen.tsx

```diff
@@ -239,6 +239,26 @@ const ErrorsScreen = (_props: Props) => {
           />
         ) : null}
         <View style={styles.mainViewBottomWhiteSpace} />
+        <Button
+          title="Log console assert"
+          onPress={() => {
+            // Sample code from https://gitlab.cin.ufpe.br/vrs2/iot-trafficlight-final/-/blob/main/node_modules/event-target-shim/dist/event-target-shim.mjs?ref_type=heads#L40
+            // This code is used by Metro and the assert is triggered when interacting with Debug Symbolicator integration.
+            // When using consoleintegration and filtering assert, it could trigger an infinite loop where debug symbolicator
+            // triggers the console integration, then it creates an event, that again calls debug symbolicator and the loop restart.
+            function pd(event: string) {
+              // const retv = privateData.get(event);
+              const retv = null;
+              console.assert(
+                  retv != null,
+                  "'this' is expected an Event object, but got",
+                  event
+              );
+              return retv;
+          }
+            pd('<object>');
+          }}
+        />
```

Comment:
l: I noticed a minor UI issue in the sample were the test button hidden below the tab bar. 

```suggestion
        />
        <View style={styles.mainViewBottomWhiteSpace} />
```

<img src="https://github.com/user-attachments/assets/ec5f3f1a-4fbe-4a56-95f2-9a49468ef01f" width=360 />

Reply:
Thank you for the suggestion! I thought it was my odd emulator resolution